packages/main/src/DateRangePicker.js

```diff
@@ -249,7 +249,18 @@ class DateRangePicker extends DatePicker {
 	}
 
 	_splitValueByDelimiter(value) {
-		return value.split(this._effectiveDelimiter).map(date => date.trim()); // just split by delimiter and trim spaces
+		let valuesArray = [];
+
+		if (this.getFormat().oFormatOptions.pattern.indexOf(this._effectiveDelimiter) >= 0) {
+			const midIndex = Math.round(value.length / 2);
+
+			valuesArray[0] = value.slice(0, midIndex).split(this._effectiveDelimiter).join("").trim();
```

Comment:
.split(this._effectiveDelimiter).join("") -> this will remove all "-"s from the first part
same for the second part
Try running the function with "2020/01/01 - 2021/02/02" f.e. 
You'll end up with "20200101" and "20210202" respectively.

IMO just get the mid index, everything from 0 to midIndex - 1 is the first part, and after mid index the second.

Reply:
This is done, because "2020/01/01 -" can't be formatted with our formatter, but 20200101 can. That's why i am removing the symbols that are breaking the formatting.